### PR TITLE
Share + Delete from Make

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Related #2018

File: www_src/pages/make/make.jsx [hunk 1]
```diff
@@ -98,11 +99,38 @@ var Make = React.createClass({
...
+            }, (err, body) => {
+              if (err) {
+                return this.onError(err);
+              }
+
+              console.warn('Deleted project: ' + e.projectID);
+              this.load();
+            });
+          } else if (event.label === 'Share') {
+            // TODO
```
Comment (gvn) [inline www_src/pages/make/make.jsx]: Connect to JS->Android bridge method.

File: www_src/components/card/card.jsx [hunk 2]
```diff
@@ -12,8 +17,15 @@ var Card = React.createClass({
...
 
         <div className="meta">
-          <div className="title">{this.props.title}</div>
-          <div className="author">{this.props.author}</div>
+          <div className="text">
+            <div className="title">{this.props.title}</div>
+            <div className="author">{this.props.author}</div>
+          </div>
+          <div className="action" hidden={!this.props.showButton}>
+            <button onClick={this.actionsClicked}>
```
Comment (k88hudson) [inline www_src/components/card/card.jsx]: This is slightly visually off-centre

File: www_src/pages/make/make.jsx [hunk 3]
```diff
@@ -98,11 +99,41 @@ var Make = React.createClass({
     }
   },
 
+  cardActionClick: function (e) {
+    dispatcher.fire('modal-switch:show', {
+      config: {
+        actions: ['Share', 'Delete'],
+        callback: (event) => {
+          if (event.label === 'Delete') {
+            api({
```
Comment (k88hudson) [inline www_src/pages/make/make.jsx]: can you `setState({loading: true})` and `false` to block user interactions here?

**Comment by k88hudson** [comment]
Couple of small things I noticed, looks good to go otherwise! R+ [[unflagged]](http://plzkthx.herokuapp.com)

**Pull request merged**